Setup/UI/WixUI.wxs

```diff
@@ -50,20 +50,10 @@
       <Publish Dialog="GitEx_InstallDirDlg" Control="Next" Event="SpawnDialog" Value="InvalidDirDlg" Order="3"><![CDATA[NOT WIXUI_DONTVALIDATEPATH AND WIXUI_INSTALLDIR_VALID<>"1"]]></Publish>
       <Publish Dialog="GitEx_InstallDirDlg" Control="Next" Event="NewDialog" Value="GitEx_CustomizeDlg" Order="4">WIXUI_DONTVALIDATEPATH OR WIXUI_INSTALLDIR_VALID="1"</Publish>
 
-      <Publish Dialog="GitEx_CustomizeDlg" Control="Back" Event="NewDialog" Value="GitEx_MaintenanceTypeDlg" Order="1">Installed</Publish>
-      <Publish Dialog="GitEx_CustomizeDlg" Control="Back" Event="NewDialog" Value="GitEx_InstallDirDlg" Order="2">NOT Installed</Publish>
-      <Publish Dialog="GitEx_CustomizeDlg" Control="Next" Event="NewDialog" Value="GitEx_VerifyReadyDlg" Order="1">Installed</Publish>
```

Comment:
@Johno-ACSLive Do you know how this should be handled, currently "NOT Installed" is always selected (this PR is postponed, just changing the behavior at upgrades)

Reply:
Let me look into this, there are a few rules that impact behaviour

Reply:
I just cloned from master branch and built the installer, it seems to ask all the questions it would on fresh install.

The installer is detecting the current version, if i leave the version attributes to 3.1.0 it complains but if i set to 3.2.0 i can see it checks and then moves on. This suggests that the installer detection result is not being passed onto later stages of the process (or for some reason they are ignoring the result).

Let me dig a little further, I'm not a fan of WiX 😜 

Creating installers should be a breeze (well it usually is with other tools 😃)

I have looked at porting to Advanced Installer, the WiX project fails to convert though. It seems to allow me to continue but I don't really trust the results. I have reported this to the Advanced Installer team, they have been able to reproduce the issue and hopefully will resolve it.

Anyway that's a little off topic.